<launch>
  <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
  <!-- The name of the parameter under which the URDF is loaded -->
  <arg name="robot_description" default="robot_description"/>
  <arg name="plugin" default="tesseract_collision/BulletContactChecker"/>
  <arg name="joint_state_topic" default=""/>
  <arg name="environment_topic" default=""/>
  <arg name="published_environment_topic" default=""/>

  <node pkg="tesseract_monitoring" type="tesseract_monitoring_environment_node" name="tesseract_monitoring_environment">
    <param name="robot_description" type="string" value="$(arg robot_description)"/>
    <param name="plugin" type="string" value="$(arg plugin)"/>
    <param name="joint_state_topic" type="string" value="$(arg joint_state_topic)"/>
    <param name="environment_topic" type="string" value="$(arg environment_topic)"/>
    <param name="published_environment_topic" type="string" value="$(arg published_environment_topic)"/>
  </node>
</launch>
